### PR TITLE
Website updates2023

### PR DESCRIPTION
**Pull request by fiskanibanda**
Final Dissertation posts - @vukosim

### CODE REVIEW

File: _posts/2023-03-06-Andani-dissertation.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+### Supervisor(s)
+
+Prof. V. Marivate, Dr. M. Adendorff
+
+## Abstract
+
+South Africa has eleven official languages and amongst the eleven languages only 9 languages are local low-resourced languages. As a result, it is essential to build the resources for these languages so that they can benefit from advances in the field of natural language processing. In this project, the focus was to create annotated datasets for the isiZulu and siSwati local languages based on news topic classification tasks and present the findings from these baseline classification models. Due to the shortage of data for these local South African languages, the datasets that were created were augmented and oversampled to increase data size and overcome class classification imbalance. In total, four different classification models were used namely Logistic regression, Naive bayes, XGBoost and LSTM. These models were trained on three different word embeddings namely Count vectorizer, TFIDF vectorizer and word2vec. The results of this study showed that XGBoost, Logistic regression and LSTM, trained from word2vec performed better than the other combinations.
+
+## Publications
```
Comment (vukosim) [inline _posts/2023-03-06-Andani-dissertation.md:20]: See publication 

Andani Madodonga, Vukosi Marivate, and Matthew Adendorff. Izindaba-Tindzaba: Machine learning news categorisation for Long and Short Text for isiZulu and Siswati, *Journal of the Digital Humanities Association of Southern Africa *, 2023. [NLP] <> [[Paper URL](https://upjournals.up.ac.za/index.php/dhasa/article/view/4449)] DOI: [doi.org/10.55492/dhasa.v4i01.4449](https://dx.doi.org/doi.org/10.55492/dhasa.v4i01.4449)

File: _posts/2023-03-06-Yolanda-dissertation.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Publications
```
Comment (vukosim) [inline _posts/2023-03-06-Yolanda-dissertation.md:20]: Maybe also embedd this video https://www.youtube.com/watch?v=GO3k72h9i0Q

**Review by vukosim — CHANGES_REQUESTED** [review]
See my comments

File: _posts/2023-03-06-Yolanda-dissertation.md (hunk 2 above)
Comment (fiskanibanda) [reply _posts/2023-03-06-Yolanda-dissertation.md:24]: Added

File: _posts/2023-03-06-Andani-dissertation.md (hunk 1 above)
Comment (fiskanibanda) [reply _posts/2023-03-06-Andani-dissertation.md:20]: Added

**Review by vukosim — APPROVED** [review]

**Pull request merged**